source/docs/software/kinematics-and-odometry/mecanum-drive-kinematics.rst

```diff
@@ -35,9 +34,25 @@ The ``MecanumDriveKinematics`` class accepts four constructor arguments, with ea
         m_frontLeftLocation, m_frontRightLocation, m_backLeftLocation,
         m_backRightLocation};
 
+   .. code-block:: python
+
+      from wpimath.geometry import Translation2d
+      from wpimath.kinematics import MecanumDriveKinematics
+
+      # Locations of the wheels relative to the robot center.
+      m_frontLeftLocation = Translation2d(0.381, 0.381)
```

Comment:
Python should not use `m_` notation. Do `self.name` or just `name` if not in a function.